test/test_examples.sh

```diff
@@ -5,53 +5,63 @@
 
 set -e
 
-WORKSPACE=$1
+SOURCE_DIR=$1
 BUILD_DIR=$2
 INSTALL_DIR=$3
+CMAKE_INSTALL_PREFIX=$4
 
 echo "Running: $0 $*"
 
 function print_usage() {
 	echo "$(basename $0) - test all examples standalone"
-	echo "Usage: $(basename $0) <workspace_dir> <build_dir> <install_dir> <list-of-examples-to-run>"
+	echo "Usage: $(basename $0) <source_dir> <build_dir> <install_dir> <CMAKE_INSTALL_PREFIX> <list-of-examples-to-run>"
 }
 
+# CMAKE_INSTALL_PREFIX can be the empty string ""
 if [ "$3" == "" ]; then
 	print_usage
 	echo -e "Error: too few arguments\n"
 	exit 1
 fi
 
-if [ "$4" == "" ]; then
+if [ "$5" == "" ]; then
 	print_usage
 	echo "No examples to run!"
 	exit 0
 fi
 
-if [ ! -f $WORKSPACE/README.md ]; then
-	echo -e "error: incorrect <workspace_dir>: $WORKSPACE\n"
+if [ ! -f $SOURCE_DIR/README.md ]; then
+	echo -e "error: incorrect <source_dir>: $SOURCE_DIR\n"
 	print_usage
 	exit 1
 fi
 
-WORKSPACE=$(realpath $WORKSPACE)
+[ ! -d ${INSTALL_DIR}/${CMAKE_INSTALL_PREFIX} ] && mkdir -p ${INSTALL_DIR}/${CMAKE_INSTALL_PREFIX}
```

Comment:
You can omit the first part and the following `mkdir -p` command just won't do anything if the dir already exists.

Reply:
Right. Done.